src/components/CounterInput/__test__/counterInput.spec.js

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import { mount } from 'enzyme';
+import CounterInput from '../';
+
+describe('<CounterInput />', () => {
+    it('should mount a input type number with a value of 5', () => {
+        const component = mount(<CounterInput value={5} />);
+        const input = component.find('input');
+        expect(input.prop('type')).toBe('number');
+        expect(input.prop('value')).toBe(5);
+    });
+
+    it('should onChange have been called with -1 when click in minusButton and 1 when click in plusButton', () => {
```

Comment:
here there are two test, on for the minusButton and other for plusButton, as a best practice for unit testing we should try to make only one expect for test, although sometimes we don't this for convenience reasons
also the name of this type of test normally is:
```should fire onChange with ...```